Lesson6/src/Animals.java

```diff
@@ -0,0 +1,21 @@
+public abstract class Animals {
```

Comment:
```suggestion
public abstract class Animal {
```
класс подразумевает описание одного животного, когда будет создавать от него объекты